symforce/geo/rot2.py

```diff
@@ -137,6 +137,14 @@ def from_angle(cls, theta: T.Scalar) -> Rot2:
         """
         return cls.from_tangent([theta])
 
+    def to_angle(self, epsilon=sf.epsilon()) -> T.Scalar:
+        """
+        Get the angle of this Rot2 in radians
+
+        This is equivalent to ``to_tangent()[0]``
+        """
+        return sf.atan2(self.z.imag, self.z.real, epsilon=epsilon)
```

Comment:
I think this should just call `to_tangent` to avoid duplicating the implementation, like `from_angle` does with `from_tangent`

Reply:
That makes sense, I changed it now.